### PR TITLE
Display version number in settings

### PR DESCRIPTION
**Pull request by micedreams**
Screenshot 2023-09-29 at 18 58 45

Screenshot 2023-09-29 at 18 59 00

### CODE REVIEW

File: lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import 'package:package_info_plus/package_info_plus.dart';
+
+class VersionNumberCubit extends Cubit<String> {
+  VersionNumberCubit() : super('');
+
+  void getVersion() async {
+    final info = await PackageInfo.fromPlatform();
+    emit(info.version);
+  }
+}
```
Comment (SankethBK) [inline lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart:11]: Creating a new cubit is not necessary as we don't have any dynamic properties in the state. Also, we can't update the version number dynamically in Flutter. So It's fine to call `PackageInfo.fromPlatform();` directly in the `VersionNumber` widget.

**Review by SankethBK — COMMENTED** [review]
Minor comment, rest all looks good!

File: lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart (hunk 1 above)
Comment (micedreams) [reply lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart:11]: @SankethBK I did not see Future Builder being used anywhere... that's why I created a cubit. Can I use a Future builder? or should I use something else?!!

Comment (micedreams) [reply lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart]: Hey again... I went ahead and used Future builder.

Comment (SankethBK) [reply lib/features/auth/presentation/bloc/cubit/version_number_cubit.dart]: Hey `FutureBuilder` is fine

**Review by SankethBK — APPROVED** [review]

**Comment by SankethBK** [comment]
Thank you again for contribution

**Pull request merged**